tests/e2e/mgmt_wrkld_domain_isolation_negative.go

```diff
@@ -0,0 +1,485 @@
+
+		ginkgo.By("Creating statefulset")
+		sts := createCustomisedStatefulSets(ctx, client, namespace, true, replicas, false, nil,
+			false, true, "", "", storageclassZ2, storagePolicyNameZ2)
+
+		ginkgo.By("Mark zone-2 for removal from wcp namespace")
+		err = markZoneForRemovalFromWcpNs(vcRestSessionId, namespace,
+			zone2)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		ginkgo.By("Wait for PVC to reach Bound state.")
+		_, err = fpv.WaitForPVClaimBoundPhase(ctx, client,
+			[]*v1.PersistentVolumeClaim{pvclaim1}, framework.ClaimProvisionTimeout)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		allowedTopologies = setSpecificAllowedTopology(allowedTopologies, topkeyStartIndex, 1,
+			2)
+		err = verifyPvcAnnotationPvAffinityPodAnnotationInSvc(ctx, client, sts, nil, nil, namespace,
+			allowedTopologies)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		ginkgo.By("Perform scaling operation on statefulset. Increase the replica count to 4 when zone is marked for removal")
+		ginkgo.By("Scaleup any one StatefulSets replica")
+		_, scaleupErr := scaleStatefulSetPods(client, sts, 4)
+		gomega.Expect(scaleupErr).NotTo(gomega.HaveOccurred())
+
+		// Validate pods are in Pending state
+		pods, err := fss.GetPodList(ctx, client, sts)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		for _, pod := range pods.Items {
+			if pod.Name == "web-3" {
+				if pod.Status.Phase != v1.PodPending {
+					framework.Failf("Expected pod to be in: %s state but is in: %s state", v1.PodPending,
+						pod.Status.Phase)
+				}
+			}
+		}
+	})
+
+	/*
+		Testcase 10 - Neg scenario
+		Mark zone for removal when WCP is down, create new STS/Deployment/VMService on the same zone before recovering WCP
+
+		Test steps:
+		1. Create a wcp namespace and add zone-2 and zone-3 to it.
+		Assign shared storage policy, zonal2 and zonal3 policy to this ns
+		2. Create volumes, pods and statefulsets.
+		3. Verify volumes created successfully and Pods reach running state.
+		4. Perform wcp service down and at the same time mark zone2 for removal.
+		5. Perform scaling operation when zone2 is marked for removal and wcp service is down.
+		6. Now create new pvc with requested topology of the zone2 which is marked for removal
+		7. Volume creation should fail with an appropriate error message
+		8. Create new volumes when wcp service is down and zone2 is marked for removal
+		9. Create deployment, VM Service VM and attach it to the volume created in step #6
+		10. Expecting that all these workloads should get created when wcp service is down.
+		11. Now recover wcp.
+		12. Validate the data
+		13. If workload creation fails, valid the error.
+		14. If it passes, perform cleanup.
+	*/
+	ginkgo.It("Workload creation when WCP is down", func() {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		// flag to check the WCP statu
+		isWcpServicestopped := false
+
+		storagePolicyNameZ1 := GetAndExpectStringEnvVar(envZonal1StoragePolicyName)
```

Comment:
give comment what all policies you are reading, this applies to all the testcases